### PR TITLE
Add IntelliJ IDEA Ultimate (EAP)

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19857)

### CODE REVIEW

**Comment by vedantmgoyal2009** [comment]
anyone please help me, my #19859 is successful but this PR is failing

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll have to update the version folder too.

File: manifests/j/JetBrains/IntelliJIDEA/Ultimate/EAP/212.4535.1/JetBrains.IntelliJIDEA.Ultimate.EAP.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Tags: 
+ - "ide"
+ - "java"
+ - "jetbrains"
+ - "intellij"
+ShortDescription: "IntelliJ IDEA: The Capable & Ergonomic Java IDE by JetBrains. Note, however, that because EAPs are always works in progress, some of their features might not always work as expected."
+Installers:
+ - Architecture: "x64"
+   InstallerType: "nullsoft"
+   InstallerUrl: "https://download.jetbrains.com/idea/ideaIU-212.4535.15.exe"
```
Comment (ItzLevvie) [inline manifests/j/JetBrains/IntelliJIDEA/Ultimate/EAP/212.4535.1/JetBrains.IntelliJIDEA.Ultimate.EAP.yaml:19]:
```suggestion
   InstallerUrl: "https://download-cdn.jetbrains.com/idea/ideaIU-212.4535.15.exe"
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/j/JetBrains/IntelliJIDEA/Ultimate/EAP/212.4535.1/JetBrains.IntelliJIDEA.Ultimate.EAP.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+PackageIdentifier: "JetBrains.IntelliJIDEA.Ultimate.EAP"
+PackageVersion: 212.4535.15
+PackageLocale: "en-US"
+Publisher: "JetBrains s.r.o."
+PackageName: "IntelliJ IDEA Ultimate (EAP)"
```
Comment (ItzLevvie) [inline manifests/j/JetBrains/IntelliJIDEA/Ultimate/EAP/212.4535.1/JetBrains.IntelliJIDEA.Ultimate.EAP.yaml:5]:
```suggestion
PackageName: "IntelliJ IDEA Ultimate Edition (EAP)"
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/j/JetBrains/IntelliJIDEA/Ultimate/EAP/212.4535.15/JetBrains.IntelliJIDEA.Ultimate.EAP.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ - "ide"
+ - "java"
+ - "jetbrains"
+ - "intellij"
+ShortDescription: "IntelliJ IDEA: The Capable & Ergonomic Java IDE by JetBrains. Note, however, that because EAPs are always works in progress, some of their features might not always work as expected."
+Installers:
+ - Architecture: "x64"
+   InstallerType: "nullsoft"
+   InstallerUrl: "https://download-cdn.jetbrains.com/idea/ideaIU-212.4535.15.exe"
+   InstallerSha256: e5c67d1decb0188bc3488fd164a96e71674f10786ada21d9812e86c438299fdf
```
Comment (ItzLevvie) [inline manifests/j/JetBrains/IntelliJIDEA/Ultimate/EAP/212.4535.15/JetBrains.IntelliJIDEA.Ultimate.EAP.yaml:20]:
```suggestion
   InstallerSha256: e5c67d1decb0188bc3488fd164a96e71674f10786ada21d9812e86c438299fdf
   ProductCode: "IntelliJ IDEA 212.4535.15"
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**